src/customDeclarations.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module "*.png";
+declare module "*.svg";
```

Comment:
also each module should be in its own declaration file - in this case you can have one for png and one for svg